layers/+tools/eaf/packages.el

```diff
@@ -48,8 +48,32 @@
     :defer t
     :init
     (progn
+      (with-eval-after-load 'eaf
```

Comment:
You can rewrite
```elisp
(list 'foo 'bar)
```
as
```elisp
'(foo bar)
```

It's much simpler.

Reply:
Thank you so much, I will modify based on this, it is valuable for me as elisp rookie 😄